addons/web/static/src/views/kanban/kanban_record.js

```diff
@@ -308,10 +308,12 @@ export class KanbanRecord extends Component {
                 const listOrGroup = group || list;
                 if (listOrGroup.deleteRecords) {
                     this.dialog.add(ConfirmationDialog, {
-                        body: env._t("Are you sure you want to delete this record?"),
+                        title: env._t("Bye-bye, record!"),
```

Comment:
You should think twice about this one... Deleting a record is irreversible, and the new message is totally unclear.